lib/phoenix_pubsub_redis/redis_server.ex

```diff
@@ -67,6 +70,11 @@ defmodule Phoenix.PubSub.RedisServer do
     {:noreply, state}
   end
 
+  def handle_info({:redix_pubsub, redix_pid, :disconnected, %{reason: reason}}, %{redix_pid: redix_pid} = state) do
+    Logger.error "disconnected from pubsub: #{reason}"
+    {:noreply, state}
```

Comment:
Should we update the state to say we are no longer connected or we don't care because redix will take care of automatically reconnecting?